serializers/config.go

```diff
@@ -0,0 +1,32 @@
+package serializers
+
+type Config struct {
+	ConnectionConfiguration ConnectionConfiguration
+	UsersConfiguration []UsersConfiguration
+	ChannelsConfiguration []ChannelsConfiguration
+	PostsConfiguration PostsConfiguration
+}
+
+type ConnectionConfiguration struct {
+	ServerURL string 
+	WebSocketURL string 
+	AdminEmail string
+	AdminPassword string 
+}
+
+type UsersConfiguration struct {
+	Username string 
+	Password string 
+	Email string
+}
+
+type ChannelsConfiguration struct {
+	DisplayName string 
+	Name string 
+	Type string
+	TeamId string
```

Comment:
```suggestion
	TeamID string
```

Reply:
this is already fixed in next PR https://github.com/Brightscout/mattermost-load-test-scripts/blob/d609d6a50388f19659a6be582431f4008c7d81a3/config/config.sample.json#L25